### PR TITLE
Advanced First Aid Kit

### PR DESCRIPTION
**Pull request by Steelpoint**
This PR adds a new 'Advanced First Aid Kit', for advanced injuries or wounds. Advanced!

(Credit to Anonmare for the idea; https://tgstation13.org/phpBB/viewtopic.php?f=9&t=20309&view=unread#p453859)

screenshot 2018-11-13 12 46 43
screenshot 2018-11-14 23 59 48

The Adv Kit can be bought from a Medical Vendor for $600 currently. It contains...
* 3x Synthflesh Patches - Described ingame as a 'Synthflesh Patch' that heals both injuries, with a unique sprite.
* 2x Atropine Autoinjectors - Intended to help heal people out of a critical state
* 1x Gauze
* 1x Pill bottle of Penetic Acid - To help purge radiation damage and toxins.

:cl: Steelpoint
add: A new advanced first aid kit is now purchasable from any approved medical vendor. The kit contains an array of advanced healing items for more complex wounds and injuries.
/:cl:

### CODE REVIEW

File: code/modules/reagents/reagent_containers/patch.dm [hunk 1]
```diff
@@ -37,4 +37,10 @@
...
 	desc = "Helps with burn injuries."
 	list_reagents = list("silver_sulfadiazine" = 20)
-	icon_state = "bandaid_burn"
\ No newline at end of file
+	icon_state = "bandaid_burn"
+
+/obj/item/reagent_containers/pill/patch/synthflesh
+	name = "brute and burn patch"
+	desc = "Helps with brute and burn injuries."
+	list_reagents = list("synthflesh" = 15)
```
Comment (swindly) [inline code/modules/reagents/reagent_containers/patch.dm]: This heals less than 40 damage whereas brute and burn patches heal over 100.

Comment (Steelpoint) [reply code/modules/reagents/reagent_containers/patch.dm]: Well I can up the dosage.

**Comment by 81Denton** [comment]
I'd name them synthflesh patches, otherwise people will assume it's just a patch with 25u styptic+25u silver sulf. After all, synthflesh can be used to fix up corpses too.

**Comment by Steelpoint** [comment]
You can use em to heal corpses?

**Comment by 81Denton** [comment]
Yeah, it's super useful for when you have a defib or strange reagent handy.

**Comment by Steelpoint** [comment]
I'll rename it then

**Comment by XDTM** [comment]
I'd suggest putting advanced health analyzers as separate items, since they're pretty much the ideal premum item by themselves, and adding something that heals toxin damage in its place.

**Comment by Borisvanmemes** [comment]
The adv health analyzer is a CMO item iirc, plus it's kinda too good to also just be tossed in something else thats also good.

**Comment by Steelpoint** [comment]
You can print them off from the techfabs already, nonetheless I will remove it and replace it with the antitoxin.

**Comment by Nickkoranda** [comment]
If I'm spending $600 having the adv health scanner would be really nice. It's not that much better than a normal health scanner besides showing a few minor things like brain damage.

**Comment by XDTM** [comment]
It can also bypass fakedeath

**Comment by Steelpoint** [comment]
I'll defer to a maintainer on that regard then.

**Comment by Borisvanmemes** [comment]
Oh, it's in techfabs?
lmao i'm a dumbass.

**Comment by delaron** [comment]
Anti-Toxin or syringe of Pentetic Acid?

**Comment by 81Denton** [comment]
Anti-toxin would be more appropriate since pentetic flushes all chems and rads (instead of just toxins)

**Comment by ExcessiveUseOfCobblestone** [comment]
Penetic should be fine since the only time anti-tox (aka reagent in bulk roundstart and easily craftable) wouldn't be viable is when you have a crazy amount of toxins.

The alternative now is just letting them die and clone lol.

**Comment by optimumtact** [comment]
take it out if it's already a techfab item

**Comment by Bawhoppen** [comment]
Gut reaction is this may be OP, but we can always try it out and remove it later.
Also if you could, make it the purple medkit in the sprite file instead of red? I feel like red should be used for some form of emergency medkit.

**Comment by Steelpoint** [comment]
I'll remove the adv analyzer and replace it with an appropriate anti-toxin pill bottle. 

An emergency medkit however is an interesting idea, but I don't know what it would include to not be too effective.

**Comment by Bawhoppen** [comment]
I tried to add one once and used tricordrazine injectors for it. Turns out that after Goofchem, tricord is like the most powerful healing chemical, so it didn't get merged. I had thought it was still as bad as it was in original Trekchem.

**Comment by Steelpoint** [comment]
Best I could think of is make it look smaller sprite wise, and put in a basic anti-oxyloss pill, one sheet of gauze and one piece of ointment and bandages for 1 use only.

**Comment by Steelpoint** [comment]
Ok, I removed the advance analyzer and replaced it with a pill bottle of Penetic Acid.

**Comment by ExcessiveUseOfCobblestone** [comment]
perhaps consider a single bottle instead a full set of pills lol...

**Comment by optimumtact** [comment]
peeps, it's a lot easier to buff something After the fact than it is to nerf it, can we please aim low and then move up instead of aiming high and having to drag everyone kicking and screaming back down

**Comment by FantasticFwoosh** [comment]
Its a pity you can't just insert the chemicals and some material to print off **entire** first aid kits, since the protolathes can hold reagents.

**Comment by optimumtact** [comment]
why is that a pity, it sounds absurdly overpowered.

**Comment by Steelpoint** [comment]
The PR is good to go. I reduced the pill amount to 3 pills, which should be fairer.

**Comment by improvedname** [comment]
Be cmo. Have budget id. Deconstruct one medvendor. Take stocking parts. Buy 1 advanced medkit. Refill. Buy another. Refill. Buy another.

**Comment by ExcessiveUseOfCobblestone** [comment]
yet another reason starting with an abundance of money roundstart for "rp" is dumb gameplay-wise

**Comment by Bawhoppen** [comment]
#UsePurpleMedkitSprite

**Comment by ExcessiveUseOfCobblestone** [comment]
I will be sliding into pills.dm soon, ugh!

**Comment by ExcessiveUseOfCobblestone** [comment]
I'ma go ahead and merge this but in the future I'd rather have items like the sec webbing instead of an item bypassing the medical/chem interaction.

I don't want econ to be a way for people to shortcut obstacles currently in place.

**Pull request merged**